olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1335,5 +1335,9 @@ spec:
       image: "registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-post-install:knative-v1.8"
     - name: "KAFKA_IMAGE_knative-kafka-storage-version-migrator__migrate"
       image: "registry.ci.openshift.org/openshift/knative-eventing-storage-version-migration:knative-v1.8"
+    - name: "FUNC_IMAGE_nodejs-16"
+      image: "registry.access.redhat.com/ubi8/nodejs-16"
+    - name: "FUNC_IMAGE_openjdk-17"
+      image: "registry.access.redhat.com/ubi8/openjdk-17"
```

Comment:
This is a generated file you should update either:
- script: https://github.com/openshift-knative/serverless-operator/blob/main/hack/generate/csv.sh
- template: https://github.com/openshift-knative/serverless-operator/blob/main/templates/csv.yaml